resources/nls/login.properties

```diff
@@ -116,6 +116,7 @@ password.newPassword.tooltip = New password
 password.confirmPassword.placeholder = Repeat password
 password.confirmPassword.tooltip = Repeat password
 password.error.match = New passwords must match
+recovery.sms.hint = SMS can only be used if a mobile phone number has been configured.
```

Comment:
@ravikanodia-okta fyi before checking in your code for the i18n sign-in-widget change, we'll need to copy over this file again since we are modifying it in PR's like this one. 

**Important** When you copy it over, you should do this:
1. Copy file
2. Replace the last translated sha line with the one that's currently in i18n (https://github.com/okta/npm/blob/master/i18n/properties/login.properties#L2)